programs/config/src/lib.rs

```diff
@@ -26,20 +26,6 @@ fn process_instruction(
     Ok(())
 }
 
-solana_entrypoint!(entrypoint);
-fn entrypoint(
-    program_id: &Pubkey,
-    keyed_accounts: &mut [KeyedAccount],
-    data: &[u8],
-    tick_height: u64,
-) -> Result<(), InstructionError> {
-    solana_logger::setup();
-
-    trace!("process_instruction: {:?}", data);
-    trace!("keyed_accounts: {:?}", keyed_accounts);
-    process_instruction(program_id, keyed_accounts, data, tick_height)
-}
-
 #[cfg(test)]
 mod tests {
```

Comment:
TODO: remove "tests"

Reply:
why?

Reply:
I guess I was thinking they're at the wrong level

Reply:
So now that I moved that module to the _api crate, you have the option to put BankClient tests into the integration tests.  From the other dylib crate, you couldn't `use solana_config_program` from the integration test.

I don't feel strongly about where they should be. They aren't pure unit-tests, but they don't do any networking I/O.  BankClient+Runtime is a very handy way to make sure your program follows all the runtime's rules.  Working with KeyedAccounts is a bit painful, takes some planning, and that interface might change on you.